### ISSUE TITLE
Bug🐛: macOS keyword substitution errors

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/open-horizon.github.io#281]
### Describe the bug.

https://open-horizon.github.io/docs/developing/service_operators.html

There is a keyword substitution that didn't work.
`Note: For {{site.data.keyword.macos_notm}} users, `

Let's look in this Markdown file. We see a lowercased substitution string. 
https://github.com/open-horizon/open-horizon.github.io/blob/master/docs/developing/service_operators.md

However in the keyword.yml file the substitution string for macOS is mixed case.
https://github.com/open-horizon/open-horizon.github.io/blob/master/_data/keyword.yml
```
macOS_notm:
  macOS
macOS:
  macOS&reg;
```

We should scan the docs/ tree for any other misspelling of the macOS substitution strings.
I don't see any other `{{site.data.keyword.macOS}}` mistakes, but the following markdown files contain 
unsubstituted references to `macOS` or `macOs`
```
docs/developing/service_containers.md
docs/hub/post_install.md
docs/hub/prepare_for_edge_nodes.md
docs/installing/many_install.md
docs/installing/advanced_man_install.md
docs/installing/adding_devices.md
docs/troubleshoot/index.md
```

**Comment by joewxboy** [comment]
Good call, @johnwalicki .Do we have a standard regarding naming ... for example, we should only use all-lowercase in keyword or property names?

**Comment by johnwalicki** [comment]
Closed #281 by #283
